### PR TITLE
Documentation now pulls proptype blocks dynamically

### PR DESCRIPTION
**Pull request by rcorral**
No need to manually update the docs when we change proptypes.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -76,17 +75,11 @@ After this you can pull down the latest module version from npm.
...
-4. Compile the assets for distribution:
-
-        npm run dist-src
+3. Bump the version:
 
-5. Now do the release with the proper flags, where `{NUMBER}` is the beta number you wish to release:
+        git run release-beta
 
-        npm publish ./ --tag -.beta-{NUMBER}
+4. Commit changes to origin
```
Comment (rcorral) [inline CONTRIBUTING.md]: Also made these few changes to the docs.

**Comment by mesosphere-web** [comment]
Reviewer: @mlunoe

**Comment by jfurrow** [comment]
Great idea!

**Comment by mlunoe** [comment]
@rcorral merging your own stuff?

**Comment by rcorral** [comment]
My bad...I think I got a little merge happy and didn't realize this didn't have a label.

**Pull request merged**